purchase_order_price_unchange/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module makes PO line qty change should not trigger price update.
```

Comment:
```suggestion
This module removes the quantity change trigger from the _onchange_quantity() method on the purchase order line, so that the quantity change does not wipe out the current price value.
```

Reply:
Thank you.
I changed description and regenerated readme files.